lib/alerts_viewer_web/live/alerts_to_close_live.html.heex

```diff
@@ -21,57 +21,89 @@
   </:col>
-  <:col :let={route} label="Alert Ids (sorted by duration)">
-    <%= @alerts_by_route[route] |> Enum.map(& &1.id) |> Enum.join(", ") %>
+  <:col :let={alert} label="Route Names">
+    <.informed_entity_icons entities={route_names_from_alert(alert, @bus_routes)} />
   </:col>
-  <:col :let={route} label="Median Adherence (minutes)">
-    <%= @stats_by_route
-    |> RouteStats.median_schedule_adherence(Atom.to_string(route))
-    |> seconds_to_minutes() %>
+  <:col :let={alert} label="Median Adherence (minutes)">
+    <%= alert
+    |> route_ids_from_alert()
+    |> Enum.map(fn route_id ->
+      @stats_by_route
+      |> RouteStats.median_schedule_adherence(route_id)
+      |> seconds_to_minutes()
+    end)
+    |> Enum.join(",") %>
   </:col>
-  <:col :let={route} label="Peak Adherence (minutes)">
-    <%= @stats_by_route
-    |> RouteStats.max_schedule_adherence(Atom.to_string(route))
-    |> seconds_to_minutes() %>
+  <:col :let={alert} label="Peak Adherence (minutes)">
+    <%= alert
+    |> route_ids_from_alert()
+    |> Enum.map(fn route_id ->
+      @stats_by_route
+      |> RouteStats.max_schedule_adherence(route_id)
+      |> seconds_to_minutes()
+    end)
+    |> Enum.join(",") %>
   </:col>
-  <:col :let={route} label="Median Instantaneous Headway (minutes)">
-    <%= @stats_by_route
-    |> RouteStats.median_instantaneous_headway(Atom.to_string(route))
-    |> seconds_to_minutes() %>
+  <:col :let={alert} label="Median Instantaneous Headway (minutes)">
+    <%= alert
+    |> route_ids_from_alert()
+    |> Enum.map(fn route_id ->
+      @stats_by_route
+      |> RouteStats.median_instantaneous_headway(route_id)
+      |> seconds_to_minutes()
+    end)
+    |> Enum.join(",") %>
   </:col>
-  <:col :let={route} label="Median Headway Deviation (minutes)">
-    <%= @stats_by_route
-    |> RouteStats.median_headway_deviation(Atom.to_string(route))
-    |> seconds_to_minutes() %>
+  <:col :let={alert} label="Median Headway Deviation (minutes)">
+    <%= alert
+    |> route_ids_from_alert()
+    |> Enum.map(fn route_id ->
+      @stats_by_route
+      |> RouteStats.median_headway_deviation(route_id)
+      |> seconds_to_minutes()
+    end)
+    |> Enum.join(",") %>
   </:col>
-  <:col :let={route} label="Peak Headway Deviation (minutes)">
-    <%= @stats_by_route
-    |> RouteStats.max_headway_deviation(Atom.to_string(route))
-    |> seconds_to_minutes() %>
+  <:col :let={alert} label="Peak Headway Deviation (minutes)">
+    <%= alert
+    |> route_ids_from_alert()
+    |> Enum.map(fn route_id ->
+      @stats_by_route
+      |> RouteStats.max_headway_deviation(route_id)
+      |> seconds_to_minutes()
+    end)
+    |> Enum.join(",") %>
   </:col>
-  <:col :let={route} label="Route has Cancelled Trip">
-    <span :if={Enum.member?(@block_waivered_routes, Atom.to_string(route))}>
+  <:col :let={alert} label="Route has Cancelled Trip">
+    <span :if={
+      length(@block_waivered_routes -- route_ids_from_alert(alert)) <
```

Comment:
🤯 That's a cool implementation I wouldn't have thought of.

It might also be possible to do something along the lines of `routes_ids |> Enum.any?(&Enum.member?(@block_waivered_routes, &1)` (not real code, just spitballing).

Reply:
My other idea was casting them to mapsets but that seemed like it would be a bit longer. I might try your Enum.any idea, it's probably easier to parse than my subtraction trickery.

Reply:
Yep, the MapSet thing would work too.